.ci/Jenkinsfile

```diff
@@ -91,70 +91,6 @@ pipeline {
         }
       }
     }
-    /**
```

Comment:
L58-63 can be removed, the ones setting the `ONLY_DOCS`

`stage('Sanity checks') {` can be removed too

Reply:
https://github.com/elastic/apm-agent-ruby/pull/1366/commits/21fb88afa69b189d78ffd0d7647abfab6ecf8aeb